api/swagger.yml

```
@@ -3094,7 +3094,7 @@ paths:
       tags:
         - objects
       operationId: deleteObject
-      summary: delete object
+      summary: delete object. Missing objects will not return a NotFound error.
```

```suggestion
      summary: delete object.
```
I believe that this belongs on the descriptions of 204 and of 404, and in a positive not a negative.

```yaml
        404:
          description: Repository or branch do not exist.
          $ref: "#/components/responses/NotFound"
```

(The same for deleteObjects, of course!)

---

Tried this one first. The description is part of the definition. Set it also on the status code - the code generators just ignore it.